samples/cellular/sms/src/main.c

```diff
@@ -78,6 +80,25 @@ int main(void)
 		return 0;
 	}
 
+	/* Setting SMS service center number if set into the configuration,
+	 * or if it's left empty, an information text is printed.
```

Comment:
Is this comment correct? The code does not print anything if the service center number is empty.

Reply:
Copy-paste. Fixed.